src/library/coverart.h

```diff
@@ -33,10 +33,13 @@ class CoverInfoRelative {
 
     // all-default memory management
     CoverInfoRelative(const CoverInfoRelative&) = default;
-    CoverInfoRelative(CoverInfoRelative&&) = default;
     CoverInfoRelative& operator=(const CoverInfoRelative&) = default;
-    CoverInfoRelative& operator=(CoverInfoRelative&&) = default;
     virtual ~CoverInfoRelative() = default;
+// Visual Studio does not support default generated move constructors
+#if !defined(_MSC_VER)
```

Comment:
>  For VS2017 the workaround is disabled and we will notice early if it works or not.

If you're up for compile failures in the future, I'll be happy to include that version statement.